### DISCUSSION TITLE
Building timeout!!

### DISCUSSION THREAD

**Discussion by julianblanco00** [vercel/community#479] [Help]
Hi! I'm trying to deploy my Nextjs site, but the build gets stuck in the message: 'Creating an optimized production build...' and then I receive a timeout error. How can I solve this? All of my pages build in just 5 minutes.

**Comment by leerob** [comment]
Does your build work locally? It's possible the timeout is occurring while trying to retrieve data from some external location.

**Comment by julianblanco00** [comment]
Hi, I cloud make the build work. But the deploy on vercel get stuck in the
message 'Deploying build outputs'
Please, I need to deploy my site but I just can't, the build is done in
just 5 minutes but I receive a timeout error when deploying

**Comment by rajivm** [comment]
Having this issue as well - keeps getting stuck on "Deploying build outputs'"

**Reply by leerob** [reply]
https://github.com/vercel/vercel/discussions/7425#discussioncomment-2159720

**Reply by rajivm** [reply]
It does work locally. Also, this is timing out after all the data fetching / actual building is complete.

**Comment by julianblanco00** [comment]
Ok thank you for that, I didn’t see it.
The problem is this:
In mi next.config.js I write some files with information to fill my site.
But the pages built with ISR can’t read those files because they think that
the files don’t exist. Is there a way to solve this?

**Comment by julianblanco00** [comment]
Yes, it's a Vercel's problem because it does not give enough time to deploy
all de output folder
